x-pack/metricbeat/module/sql/query/_meta/data.json

```diff
@@ -10,373 +10,497 @@
mance_schema_memory_classes_lost\":0,\"performance_schema_metadata_lock_lost\":0,\"performance_schema_mutex_classes_lost\":0,\"performance_schema_mutex_instances_lost\":0,\"performance_schema_nested_statement_lost\":0,\"performance_schema_prepared_statements_lost\":0,\"performance_schema_program_lost\":0,\"performance_schema_rwlock_classes_lost\":0,\"performance_schema_rwlock_instances_lost\":0,\"performance_schema_session_connect_attrs_longest_seen\":106,\"performance_schema_session_connect_attrs_lost\":0,\"performance_schema_socket_classes_lost\":0,\"performance_schema_socket_instances_lost\":0,\"performance_schema_stage_classes_lost\":0,\"performance_schema_statement_classes_lost\":0,\"performance_schema_table_handles_lost\":0,\"performance_schema_table_instances_lost\":0,\"performance_schema_table_lock_stat_lost\":0,\"performance_schema_thread_classes_lost\":0,\"performance_schema_thread_instances_lost\":0,\"performance_schema_users_lost\":0,\"prepared_stmt_count\":0,\"queries\":50,\"questions\":1,\"replica_open_temp_tables\":0,\"rsa_public_key\":\"-----BEGIN PUBLIC KEY-----\\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwFTV5dQI5FC8cSJfjteN\\nF/+ERIuwUxDLs4eErzSLl+UbWpejuBO8vEry0q5WM7sLhSRA0oyEgDVCca4woRzH\\nDxpQgpzSz1pHujTORcN3JqkJ0OKFNtKf1aYVRD12xi/modl+Wk4/myyrIY46U+sl\\n3nTjcn6Pz52J7WiowYXdHtiIWrDCVJBR2biq5rOv6BIfHY8Gg+/KOlG70aNzuB1/\\n+h9FdbGuTSZD9DPewerSMpNDGN1bg8sIe3ewEE4AOr1HWeD5O1pOALsDtHIZZLip\\nlPEgmRLMLgtak0+GMIId2VxZIevOZtV+7q5IDZR+0c4Fp16DbATFveoASrXSGGPT\\n4QIDAQAB\\n-----END PUBLIC KEY-----\\n\",\"secondary_engine_execution_count\":0,\"select_full_join\":0,\"select_full_range_join\":0,\"select_range\":0,\"select_range_check\":0,\"select_scan\":0,\"slave_open_temp_tables\":0,\"slow_launch_threads\":0,\"slow_queries\":0,\"sort_merge_passes\":0,\"sort_range\":0,\"sort_rows\":0,\"sort_scan\":0,\"ssl_accept_renegotiates\":0,\"ssl_accepts\":0,\"ssl_callback_cache_hits\":0,\"ssl_cipher\":\"\",\"ssl_cipher_list\":\"\",\"ssl_client_connects\":0,\"ssl_connect_renegotiates\":0,\"ssl_ctx_verify_depth\":18446744073709552000,\"ssl_ctx_verify_mode\":5,\"ssl_default_timeout\":0,\"ssl_finished_accepts\":0,\"ssl_finished_connects\":0,\"ssl_server_not_after\":\"May  8 07:21:23 2032 GMT\",\"ssl_server_not_before\":\"May 11 07:21:23 2022 GMT\",\"ssl_session_cache_hits\":0,\"ssl_session_cache_misses\":0,\"ssl_session_cache_mode\":\"SERVER\",\"ssl_session_cache_overflows\":0,\"ssl_session_cache_size\":128,\"ssl_session_cache_timeout\":300,\"ssl_session_cache_timeouts\":0,\"ssl_sessions_reused\":0,\"ssl_used_session_cache_entries\":0,\"ssl_verify_depth\":0,\"ssl_verify_mode\":0,\"ssl_version\":\"\",\"table_locks_immediate\":6,\"table_locks_waited\":0,\"table_open_cache_hits\":0,\"table_open_cache_misses\":0,\"table_open_cache_overflows\":0,\"tc_log_max_pages_used\":0,\"tc_log_page_size\":0,\"tc_log_page_waits\":0,\"threads_cached\":0,\"threads_connected\":1,\"threads_created\":1,\"threads_running\":2,\"uptime\":220,\"uptime_since_flush_status\":220}"
```

Comment:
This is not correct. You must not store a full JSON in elasticsearch as a plain text. This must be stored as JSON too.

Reply:
ok, I was exploring to have a text output, which can be default field mapped.

Since the raw data is already in json metrics, I could remove the "results" if it adds no additional value.

Reply:
removed results text.